recipes/requests-unixsocket/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "requests-unixsocket" %}
+{% set version = "0.1.5" %}
+{% set sha256 = "a91bc0138f61fb3396de6358fa81e2cd069a150ade5111f869df01d8bc9d294c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pbr >=1.3,<2
+    - requests >=1.1,<2
```

Comment:
I'm not familiar with how pbr works, but typically requests and urllib3 would not be required in the build section due to the `--single-version-externally-managed` flag. 

Otherwise, everything else looks good.